ehb_service/apps/api/views/subject.py

```diff
@@ -13,9 +15,8 @@
 log = logging.getLogger(__name__)
 
 
-class SubjectResource(Resource):
-    supported_accept_types = ['application/json', 'application/xml']
-    model = 'core.models.identities.Subject'
+@permission_classes((permissions.AllowAny,))
```

Comment:
Instead of 'AllowAny' should this be 'IsAuthenticated'?

The AllowAny permission class will allow unrestricted access, regardless of if the request was authenticated or unauthenticated.

This permission is not strictly required, since you can achieve the same result by using an empty list or tuple for the permissions setting, but you may find it useful to specify this class because it makes the intention explicit.

Reply:
Removed the @permissionsAllowAny header from all the classes. Created a api/authentication.py file to override Django authentication so that we can continue authenticating using the api tokens that we've created instead of switching over to Django-generated auth-tokens. 803fea7